src/client/clientmap.cpp

```diff
@@ -1245,11 +1245,6 @@ void ClientMap::updateDrawListShadow(v3f shadow_light_pos, v3f shadow_light_dir,
 {
 	ScopeProfiler sp(g_profiler, "CM::updateDrawListShadow()", SPT_AVG);
 
-	v3s16 cam_pos_nodes = floatToInt(shadow_light_pos, BS);
```

Comment:
This is unused. I traced back the commit history, and this was never used. Looks like an accidental cut-n-parte.

If I ever figure out #14239 (if that's even possible) I'm going to re-add this. Until then it's dead code.